### PR TITLE
Made instructions for Jquery waypoint 6 more clear.

### LINKED ISSUE #5463: Target the same element with multiple jQuery Selectors
This is *by far* the most frequent question I see in the main chat/receive PMs about. It comes up several times a day. The answers they provide aren't syntax/conceptual issues, it is just that many campers seem to think they are supposed to add all three classes to the last one, #target1. Not sure of the best solution-it's seems straightforward to me- but I think it may be this line that causes the confusion to some of our non-native speakers:

`Although it is possible to add multiple classes in a single .addClass() call, let's add them to the same element in three different ways.`

Maybe it is because `although` that is confusing, and stronger emphasis (such as "We *can* add multiple classes in a single..... But this time, let's add one class ...") might help? Or something else, heh.

Cheers!

### PR DESCRIPTION
**Pull request by patsul12**
Closes #5463 

Used the updated instructions in the issues, changed the test to allow the selector $("button.btn") since it is also valid.

Tested in Chrome

### CODE REVIEW

File: seed/challenges/01-front-end-development-certification/jquery.json [hunk 1]
```diff
@@ -299,16 +299,18 @@
...
+        "and by id: <code>$(\"#target1\")</code>.",
+        "Using <code>.addClass()</code>, add only one class at a time to the same element, three different ways:",
         "Add the <code>animated</code> class to all elements with type <code>button</code>.",
         "Add the <code>shake</code> class to all the buttons with class <code>.btn</code>.",
         "Add the <code>btn-primary</code> class to the button with id <code>#target1</code>."
       ],
       "tests": [
         "assert(code.match(/\\$\\s*?\\(\\s*?(?:'|\")\\s*?button\\s*?(?:'|\")/gi), 'message: Use the <code>$&#40\"button\"&#41</code> selector.');",
-        "assert(code.match(/\\$\\s*?\\(\\s*?(?:'|\")\\s*?\\.btn\\s*?(?:'|\")/gi), 'message: Use the <code>$&#40\".btn\"&#41</code> selector.');",
+        "assert(code.match(/\\$\\s*?\\(\\s*?(?:'|\")\\s*?(?:button)?\\.btn\\s*?(?:'|\")/gi), 'message: Use the <code>$&#40\".btn\"&#41</code> selector.');",
```
Comment (bugron) [inline seed/challenges/01-front-end-development-certification/jquery.json]: Why are you checking the optional existence of `button` if
1. it is already done by the assert above,
2. you need to specify only one selector per one `.addClass` method?

**Comment by patsul12** [comment]
If I am remembering correctly I added this so that the test would still accept a selector like
`$('button.btn')` as well as just `$('.btn').

While I agree it is not completely necessary, it is still valid syntax. And it was something that was brought up in the initial issue.

**Comment by BerkeleyTrue** [comment]
Please rebase this PR.

**Comment by BerkeleyTrue** [comment]
Closing as stale. Please feel free to resubmit.

Happy Coding!

**Pull request closed without merging**